tutorials/topi/intro_topi.py

```diff
@@ -0,0 +1,108 @@
+"""
+Introduction to TOPI
+====================
+**Author**: `Ehsan M. Kermani <https://github.com/ehsanmok>`_
+
+This is an introductory tutorial to TVM Operator Inventory (TOPI).
+
+TOPI provides numpy-style generic operations and schedules with higher abstractions than TVM.
+
+In this tutorial, we will see how TOPI can save from writing boilerplates code in TVM.
+"""
+from __future__ import absolute_import, print_function
+
+import tvm
+import topi
+
+######################################################################
+# Basic example
+# -------------
+# Let's revisit the sum of rows operation (equivalent to :code:`B = numpy.sum(A, axis=1)`') \
+# from :ref:`reduction` tutorial. To compute the sum of rows of a two dimensional TVM tensor A, we should
+# specify the symbolic operation as well as schedule as follows
+#
+n = tvm.var("n")
+m = tvm.var("m")
+A = tvm.placeholder((n, m), name='A')
+k = tvm.reduce_axis((0, m), "k")
+B = tvm.compute((n,), lambda i: tvm.sum(A[i, k], axis=k), name="B")
+s = tvm.create_schedule(B.op)
+
+# and to examine the IR code in human readable format, we can do
+#
+print(tvm.lower(s, [A, B], simple_mode=True))
+
+# However, for such a common operation we had to define the reduce axis ourselves as well as explicit computation with
+# :code: `tvm.compute`. Imagine for more complicated operations how much details we need to provide.
+# Fortunately, we can replace those two lines with simple :code:`topi.sum` much like :code`numpy.sum`
+#
+C = topi.sum(A, axis=1)
+ts = tvm.create_schedule(C.op)
+print(tvm.lower(s, [A, C], simple_mode=True))
+
```

Comment:
add a section about numpy style operator overloading

Reply:
Do you mean just change the section title to "numpy style operator" or something else?

Reply:
add a section, by overloading I mean broadcast add and sub, which directly overloads to +, - among tensors

Reply:
b/c in this section I'm explaining how topi reduces writing manual tvm.compute

Reply:
yes the reduce example is nice, and operator overload of broadcast add is a bit different, as you don't get that from topi.broadcast_add but instead directly use +, -, so it is worth highlighting that